scripts/release/rpm/test_rpm_in_docker.sh

```diff
@@ -5,22 +5,26 @@ set -exv
 
 export USERNAME=azureuser
 
-yum --nogpgcheck localinstall /mnt/rpm/$RPM_NAME -y
+PYTHON_PACKAGE=python39
+PYTHON_CMD=python3.9
+PIP_CMD=pip3.9
 
-yum install git gcc python3-devel -y
+dnf --nogpgcheck install /mnt/rpm/$RPM_NAME -y
```

Comment:
`yum` is deprecated. We use the latest `dnf`. Note `localinstall` is now a deprecated alias of `install`: https://dnf.readthedocs.io/en/latest/command_ref.html#install-command